src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -0,0 +1,436 @@
+  CourtOrderStepIF,
+  DocumentDeliveryIF,
+  PeopleAndRoleIF
+} from '@/interfaces'
+
+@Component({
+  components: {
+    AssociationDetails,
+    Certify,
+    CourtOrderPoa,
+    DocumentDelivery,
+    StaffPayment,
+    TransactionalFolioNumber,
+    ListPeopleAndRoles
+  }
+})
+export default class DissolutionFirm extends Mixins(DateMixin) {
+  // Global getters
+  @Getter getBusinessContact!: BusinessContactIF
+  @Getter getCertifyState!: CertifyIF
+  @Getter getCompletingPartyStatement!: CertifyStatementIF
+  @Getter getCourtOrderStep!: CourtOrderStepIF
+  @Getter getCurrentDate!: string
+  @Getter getDissolutionCustodianEmail!: string
+  @Getter getDocumentDelivery!: DocumentDeliveryIF
+  @Getter getUserEmail!: string
+  @Getter getValidateSteps!: boolean
+  @Getter isPremiumAccount!: boolean
+  @Getter isRoleStaff!: boolean
+  @Getter getFolioNumber!: string
+  @Getter getTransactionalFolioNumber!: string
+  @Getter getAddPeopleAndRoleStep!: PeopleAndRoleIF
+
+  // Global actions
+  @Action setCourtOrderFileNumber!: ActionBindingIF
+  @Action setHasPlanOfArrangement!: ActionBindingIF
+  @Action setCourtOrderValidity!: ActionBindingIF
+  @Action setCertifyState!: ActionBindingIF
+  @Action setDocumentOptionalEmail!: ActionBindingIF
+  @Action setDocumentOptionalEmailValidity!: ActionBindingIF
+  @Action setTransactionalFolioNumber!: ActionBindingIF
+  @Action setTransactionalFolioNumberValidity!: ActionBindingIF
+
+  // Enum for template
+  readonly RouteNames = RouteNames
+
+  /** Is true when the Court Order conditions are not met. */
+  get isCourtOrderInvalid (): boolean {
+    return (this.getValidateSteps && !this.getCourtOrderStep.valid)
+  }
+
+  /** Is true when the Document Delivery conditions are not met. */
+  get isDocumentDeliveryInvalid (): boolean {
+    return (this.getValidateSteps && !this.getDocumentDelivery.valid)
+  }
+
+  /** Is true when the certify conditions are not met. */
+  private get isCertifyInvalid () {
+    return this.getValidateSteps && (!this.getCertifyState.certifiedBy || !this.getCertifyState.valid)
+  }
+
+  /** Handler for Valid change event. */
+  private onIsCertified (val: boolean): void {
+    this.setCertifyState(
+      {
+        valid: val,
+        certifiedBy: this.getCertifyState.certifiedBy
+      }
+    )
+  }
+
+  /** Handler for CertifiedBy change event. */
+  private onCertifiedBy (val: string): void {
+    this.setCertifyState(
+      {
+        valid: this.getCertifyState.valid,
+        certifiedBy: val
+      }
+    )
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+#dissolution-firm-form {
+  /* Set "header-counter" to 0 */
+  counter-reset: header-counter;
+}
+
+h2::before {
+  /* Increment "header-counter" by 1 */
+  counter-increment: header-counter;
+  content: counter(header-counter) '. ';
+}
+
+#dissolution-summary .v-divider {
+  background-color: $gray1;
+}
+
+::v-deep .section-container {
+  padding: 1.5rem 1.5rem;
```

Comment:
You probably don't need this, or the `::v-deep` on line 274.

Reply:
You probably don't need ANY of the v-deep style overrides in here.

Reply:
I see you removed the `::v-deep` selector modifier... Sorry, this is not what I meant. I meant some of these CSS blocks may not be needed any more (because the shared components were updated).

Did you check this? Do you still need this CSS here? If so, let me know and I'll resolve this conversation.

Reply:
Yeah, we still need this CSS, else the design will completely change

<img width="1056" alt="Screen Shot 2022-03-17 at 10 47 56 AM" src="https://user-images.githubusercontent.com/55506249/158864036-c5eefee8-8095-4abd-86a2-9874bedd0b20.png">